__tests__/kata3.numberToReversedDigits.test.js

```diff
@@ -1,7 +1,7 @@
-const { numberToReversedDigits } = require('../src');
+const numberToReversedDigits = require('../src/kata3.numberToReversedDigits.js')
 
 describe('numberToReversedDigits', () => {
   it('returns a reversed array of the number\'s digits', () => {
-    
+    expect(numberToReversedDigits(12345)).toEqual(['5', '4', '3', '2', '1'])
```

Comment:
What would happen if this was passed an array of numbers as strings?

`['1', '2', '3', '4', '5']`